odnoklassniki-android-sdk-example/src/main/java/ru/ok/android/sdk/example/MainActivity.java

```diff
@@ -119,6 +99,43 @@ public void onError(String error) {
         });
     }
 
+    @Override
+    protected void onResume() {
```

Comment:
Documentation is required to specify why it is required to maintain listener data to activity lifecycle methods. It is a sample app)